SwiftUI_Homework/SwiftUI_Homework/ShopView.swift

```diff
@@ -0,0 +1,179 @@
+        .pickerStyle(.segmented)
+        .padding()
+    }
+    
+    private var sneakerImage: some View {
+        Image(Constants.imagesName[segmentIndex])
+            .resizable()
+            .frame(width: Constants.widthValueImage, height: Constants.heightValueImage)
+            .animation(.spring())
+    }
+    
+    private var modelDescriptText: some View {
+        Text(Constants.modelDescriptions[segmentIndex])
+            .padding(.leading, Constants.leadingValue)
+            .padding(.trailing, Constants.trailingValue)
+    }
+    
+    private var sneakerModelText: some View {
+        Text("\(Constants.sneakersText)\(Constants.models[segmentIndex])")
+            .bold()
+            .font(.title)
+            .padding(.top, Constants.topValue)
+    }
+    private var priceText: some View {
+        Text("\(Constants.priceText)\(Constants.prices[segmentIndex])")
+            .padding(.leading, Constants.leadingValue)
+            .font(.title3)
+    }
+    
+    private var buyButton: some View {
+        Button {
+            isBuyShown.toggle()
+        } label: {
+            Image(systemName: Constants.dollarsignImageName)
+            Text(Constants.buyText)
+        }
+        .font(.title3)
+        .bold()
+        .frame(width: Constants.widthValueButton, height: Constants.heightValueButton)
+        .padding(.trailing, Constants.trailingValue)
+        .alert(isPresented: $isBuyShown) {
+            Alert(title: Text(Constants.questionBuyText),
+                  primaryButton: .default(Text(Constants.yesText)),
+                  secondaryButton: .cancel(Text(Constants.cancelText)))
+        }
+    }
+    
+    private var addButton: some View {
+        Button {
+            isAddShown.toggle()
+        } label: {
+            Image(systemName: Constants.basketImageName)
+            Text(Constants.addBasketText)
+        }
+        .font(.title3)
+        .bold()
+        .frame(width: Constants.widthValueButton, height: Constants.heightValueButton)
+        .actionSheet(isPresented: $isAddShown) {
+            ActionSheet(title: Text(Constants.questionBasketText),
+                        buttons: [.default(Text(Constants.addText)),
+                                  .destructive(Text(Constants.notAddText))])
+        }
+    }
+    
+    private var shareButton: some View {
+        Button {
+            isShareShown.toggle()
+        } label: {
+            Image(systemName: Constants.arrowshapeImageName)
+            Text(Constants.shareText)
+        }
+        .font(.title3)
+        .bold()
+        .padding(.trailing, Constants.trailingValue)
+        .sheet(isPresented: $isShareShown) {
+            ActivityView(activityItems: [
+        """
+\(Constants.infoProductText)
+\(Constants.priceText) \(Constants.prices[segmentIndex])
+\(Constants.models[segmentIndex])
+\(Constants.modelDescriptions[segmentIndex])
+"""
+            ])
+        }
+    }
+}
+
+struct ContentView_Previews: PreviewProvider {
```

Comment:
```suggestion

// MARK: - ContentView_Previews
struct ContentView_Previews: PreviewProvider {
```
в первой домашке была тут марка